assets/js/_main.js

```diff
@@ -0,0 +1,12 @@
+const _anchors = document.querySelectorAll('h1, h2, h3, h4, h5, h6');
+
+// adds anchor permalinks to any Header tag that has an id
+[..._anchors].forEach(anchor => {
+  if (anchor.id) {
+    const link = document.createElement('a');
+    link.dataset.icon = '';
```

Comment:
Can we add a comment to this line to explain what's happening? Should it be rendering as a box character?